src/Symfony/Bundle/TwigBundle/Extension/ActionsExtension.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  *
- * @deprecated Deprecated in 2.2, to be removed in 3.0.
+ * @deprecated since version 2.2, to be removed in 3.0.
```

Comment:
The whole class was deprecated in 2.2. I changed the constructor argument in 2.7. But anyway, nobody should use this class and it will be removed in 3.0. See #13233